handbook/engineering/distribution/tech_stack.md

```diff
@@ -1,12 +1,12 @@
 # Distribution tech stack
 
-This document lists the technologies used by the Distribution team, what they are used for, and best practices for each within Distribution.
+This document lists the Distribution team's technologies and best practices for each within the Distribution team.
```

Comment:
What was wrong with the original description? I think the new one is a bit confusing:

* these aren't really Distribution's technologies (we just use them)
* a goal of this document should be to describe _what_ each item is used for, so that we can better choose when to use what (especially since the document arose over discussions on when to use TS), which the new description removes

Reply:
This change was driven by grammar mostly. The previous sentence was written in the passive voice and had a comma splice. I also consider "Best Practices" inclusive of _what_ something would be used for. 
Happy to revert though.